packages/riipen-ui/src/components/Input.jsx

```diff
@@ -121,6 +124,12 @@ const Input = props => {
           border-color: ${theme.palette.negative.main};
         }
 
+        .underlined {
+          background-color: transparent;
```

Comment:
Wasn't sure if we should make the background colour white or transparent (see screenshot).

Reply:
i think it looks great